metadata-service/restli-servlet-impl/src/main/java/com/linkedin/metadata/resources/entity/BatchIngestionRunResource.java

```diff
@@ -57,10 +61,10 @@
   @Nonnull
   @WithSpan
   public Task<RollbackResponse> rollback(@ActionParam("runId") @Nonnull String runId,
-      @ActionParam("dryRun") @Optional Boolean dryRun, @ActionParam("hardDelete") @Optional Boolean hardDelete) {
+      @ActionParam("dryRun") @Optional Boolean dryRun, @ActionParam("safe") @Optional Boolean safe) {
```

Comment:
this will break older clients, let's keep the old action param and just mark it deprecated, but still honor it as representing a `safe = False` in the new world.

Reply:
Would you like me to implement some sort of precedence logic to use `safe` in the absence of hardDelete so that we are consistent?